app/src/main/java/org/oppia/android/app/home/HomeItemViewModel.kt

```diff
@@ -3,4 +3,6 @@ package org.oppia.android.app.home
 import org.oppia.android.app.viewmodel.ObservableViewModel
 
 /** The root [ViewModel] for all individual items that may be displayed in home fragment recycler view. */
-abstract class HomeItemViewModel : ObservableViewModel()
+abstract class
```

Comment:
This change doesn't look correct formating. 
Also we can skip any change in this file as nothing is needed.

Reply:
Fixed it.